src/core/base.jl

```diff
@@ -154,18 +143,16 @@ function build_sets(data :: Dict{AbstractString,Any})
     gen_idxs = collect(keys(gen_lookup))
     branch_idxs = collect(keys(branch_lookup))
 
-
-    buspair_indexes = collect(Set([(i,j) for (l,i,j) in arcs_from]))
-    buspairs = buspair_parameters(buspair_indexes, branch_lookup, bus_lookup)  
+    buspair_indexes = collect(Set([a[2:3] for a in arcs_from]))
+    buspairs = buspair_parameters(buspair_indexes, branch_lookup, bus_lookup)
```

Comment:
No, there's no real reason to remove ``l`` there. I'd say it's a false positive from the linter.